imagebuildah/chroot_symlink.go

```diff
@@ -62,32 +65,118 @@ func ResolveSymLink(rootdir, filename string) (string, error) {
 		return "", errors.Wrapf(err, string(output))
 	}
 
-	// Hand back the resolved symlink, will be "" if a symlink is not found
+	// Hand back the resolved symlink, will be filename if a symlink is not found
 	return string(output), nil
 }
 
+func resolveSymlinkTimeModified() {
```

Comment:
Non-blocker: can we document this function? Having a function without argument and return values is rare.

Reply:
Added comment before function. Fixed.